geppetto-showcase/install_showcase.sh

```diff
@@ -0,0 +1,20 @@
+yarn global add yalc
```

Comment:
Dario added a `setup.sh` on project level that is almost identical. Would suggest to keep only one script and inside the geppetto-showcase folder

Reply:
Didn't realize. Removed Dario's and renamed mine. Looks like a good compromise no? :stuck_out_tongue_closed_eyes:

Reply:
💯